CUDA/template/GB_jit_kernel_cuda_colscale.cu

```diff
@@ -65,8 +80,8 @@ __global__ void GB_cuda_colscale_kernel
                 for (int64_t pdelta = threadIdx.x ; pdelta < my_chunk_size ; pdelta += blockDim.x)
                 {
                     int64_t p_final ;
-                    int64_t k = GB_cuda_ek_slice_entry (&p_final, pdelta, pfirst, Ap, anvec_sub1, kfirst, slope) ;
-                    int64_t j = GBh_A (Ah, k) ;
+                    GB_Aj_TYPE k = GB_cuda_ek_slice_entry (&p_final, pdelta, pfirst, Ap, anvec_sub1, kfirst, slope) ;
```

Comment:
On the CPU kernels I usually just upcast to int64_t scalars, rather than use the 32/64 bit type.  The CUDA kernels are different since they are JIT-only, however, so this might be OK.  But I haven't looked into this yet.  That's why I left the scalar integers as int64_t for now.

Reply:
The GB_cuda_ek_slice_entry returns an int64_t, regardless of the type of the integers in the matrix.  If I make it type-specific, it would not be "GB_Aj_TYPE" since the method could be used for any matrix, say A or B, and the type would differ.  So let's stick with int64_t scalars for now.

Reply:
Fixed